Podcast/SearchHeaderView.swift

```diff
@@ -21,9 +21,9 @@ enum searchHeaderViewType {
     var title: NSMutableAttributedString {
```

Comment:
Let's not add this header view on the main search actually.. it only makes sense to have it when you go to the search series view controller by clicking "See All"
<img width="331" alt="screen shot 2018-04-12 at 3 46 16 pm" src="https://user-images.githubusercontent.com/14966713/38700382-d97e3c58-3e68-11e8-88b4-32cfdc7b4dbe.png">

Reply:
But if there are 0 results for a series then the user won't even see the header with the See all button, should I only display it if series.count = 0?

Reply:
Nvm just saw your other comment